ckan/tests/controllers/test_group.py

```diff
@@ -424,21 +424,20 @@ def test_membership_edit_page(self, current_user, app):
         assert role_option and role_option.get('value') == 'admin'
         assert page.select_one('#username').get('value') == member['name']
 
-    @pytest.mark.usefixtures("clean_db")
```

Comment:
Just in case, copy of the comment from #7040:

This clean_db cannot be removed:

test creates a user, so it will fail if db is not initialized
user created with static name, so it will fail if you run this test multiple times